webhooks-extension/src/WebhookApp.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { Route, withRouter } from "react-router-dom";
+
+import { WebhookCreate } from './components/WebhookCreate';
+import { WebhookDisplayTable } from './components/WebhookDisplayTable';
+
+class WebhooksApp extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      // Used to show webhook created notification on table display
+      showNotification: false,
```

Comment:
`ShowNotification` is a general variable, can it be more specific so it doesn't require comment

Reply:
changing to ShowNotificationOnTable